docs/src/hooks/useViewport.js

```diff
@@ -1,7 +1,10 @@
 import { useState, useEffect } from "react";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 
 export const useViewport = () => {
-    const [width, setWidth] = useState(0);
+    const { siteConfig } = useDocusaurusContext();
+    const breakpoints = siteConfig?.customFields?.breakpoints;
+    const [width, setWidth] = useState(breakpoints.lg);
```

Comment:
setting `0` as default was causing the version dropown issue, now we are using `1100` as default value.

tried setting `window.innerWidth` but it caused the build to fail